### DISCUSSION TITLE
The difference between `clip_box` and `slice_xy`

### DISCUSSION THREAD

**Discussion by tqa236** [corteva/rioxarray#612] [General]
Hello, 

First of all, thank you very much for this awesome package!

I have a question regarding 2 quite similar methods, `clip_box` and `slice_xy`. Are they doing the same thing or there are some subtle differences? 

We used `clip_box` before but discover that `slice_xy` is much faster when (supposedly) doing the same job. 

If they are indeed different, in which situation should we use each of them? 

Best,
Anh

**Comment by snowman2** [comment]
`rio.clip_box` considers the grid cell in its entirety whereas `rio.slice_xy` only considers the centroid of the cell.
